js/library.js

```diff
@@ -0,0 +1,36 @@
+const myLibrary = [];
+const target = document.getElementById('demo');
+const targetbtn = document.getElementById('add-btn');
+const targetdisplay = document.getElementById('display');
+const alertText = document.getElementById('alertText');
+
+export {
+  myLibrary, target, targetbtn, targetdisplay, alertText,
+};
+
+function Book(booktitle, author, totalPages, read) {
+  this.booktitle = booktitle;
+  this.author = author;
+  this.totalPages = totalPages;
+  this.read = read;
+}
+
+function saveLibrary() {
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+}
+
+export function addBookToLibrary(booktitle, author, totalPages, read = true) {
```

Comment:
## Fix the book creation function.
- [ ] A user seems to be unable to do that currently, we also have some informing errors in the console we might want to take care of, do you mind working on this as well? Below is a screenshot of what the user is met with. 👇🏾 
![library](https://user-images.githubusercontent.com/22345961/111681161-01836f80-8834-11eb-88b7-1f5824a4689c.png)